src/Compilers/VisualBasic/Portable/Language/LanguageFeature.vb

```diff
@@ -89,8 +100,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 
                 Case Feature.InferredTupleNames
                     Return LanguageVersion.VisualBasic15_3
-
                 Case Else
+                    If feature.IsPrototype Then Return CType(Integer.MinValue, LanguageVersion)
```

Comment:
`Integer.MinValue` is being used a guard for prototype features like `IOperation` before throwing the `UnexpectedValueException`.